src/SFA.DAS.AssessorService.Application.Api.UnitTests/Services/AssessorValidationServiceTests.cs

```diff
@@ -123,5 +123,15 @@ public void CheckIfUkprnAlreadyTaken(bool result)
             var checkResult = _assessorValidationService.IsOrganisationUkprnTaken(1234).Result;
             Assert.AreEqual(checkResult, result);
         }
+
+        [TestCase(true)]
+        [TestCase(false)]
+        public void IsOrganisationStandardTaken(bool result)
+        {
+            _registerValidationRepository.Setup(r => r.EpaOrganisationStandardExists(It.IsAny<string>(), It.IsAny<int>()))
+                .Returns(Task.FromResult(result));
+            var checkResult = _assessorValidationService.IsOrganisationStandardTaken("EPA0001", 1).Result;
+            Assert.AreEqual(checkResult, result);
+        }
```

Comment:
How can both of those TestCases pass?

Reply:
Oh I see, the .Setup has it's return set to the expected.  Not sure this is a very good test, but not sure why.

Reply:
Would the better test be to make sure that the correct values get passed through to EpaOrganisationStandardExists? 
And check the return from EpaOrganisationStandardExists gets returned from IsOrganisationStandardTaken is the same (which i guess is what this test is doing, but it doesn't need to do both test cases).

Reply:
It follows the same pattern as the other tests in here (i.e. true / false)

I read these tests as the registerValidationRepository will return the appropraite response and thus assessorValidationService should return that response too (i.e. it bubbles up through the relevant layers in the application).